app/models/service_ansible_playbook.rb

```diff
@@ -3,7 +3,11 @@ class ServiceAnsiblePlaybook < ServiceGeneric
 
   # A chance for taking options from automate script to override options from a service dialog
   def preprocess(action, add_options = {})
-    _log.info("Override with new options: #{add_options}") unless add_options.blank?
+    unless add_options.blank?
+      _log.info("Override with new options:")
+      $log.log_hashes(add_options)
```

Comment:
@lfu You can probably add the following method to the [log proxy](https://github.com/ManageIQ/manageiq/blob/master/lib/vmdb/logging.rb#L4) to achieve `_log.log_hashes`
```ruby
  def log_hashes(h, options = {})
    VMDBLogger.log_hashes(self, h, options)
  end
```